docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -0,0 +1,246 @@
+# Zowe MultiTenancy Configuration
+
```

Comment:
Can we please include a brief description of what MultiTenancy Configuration is?